### ISSUE TITLE
create Web view without dimensions

### ISSUE THREAD

**Issue by ahmadkhedr** [fluttercommunity/flutter_webview_plugin#368]
Is there a way of creating a web view without giving it a height so that it can be able to show me the whole content inside the widget's tree without the need of inserting gestureRecognizers for scrolling inside the fixed height that I have gave to the widget ...I have used the "flutter_html" package and it does not force me to give a height to the HTML widget and the container expand to contain the data automatically and I can not find this feature in the "web_view" package....and is there any way to control the text size inside the webview?

**Comment by dvird** [comment]
@ahmadkhedr why did you close it? you have found a solution?

**Comment by ahmadkhedr** [comment]
> 
> 
> @ahmadkhedr why did you close it? you have found a solution?

No, But I found another plugin that resolved my issue...It can not be a complete solution but it solved my issue at least.
the plugin is: flutter_widget_from_html 0.2.1+1
https://pub.dev/packages/flutter_widget_from_html
And I will reopen the issue with the doubt the problem in the webview plugin will be solved soon :(
